common/styles/scss/_recordset.scss

```diff
@@ -265,6 +269,7 @@
 
     >thead>tr>th {
         border-bottom: 1px solid $table-border-color;
+        border-top: 1px solid $table-border-color !important;
```

Comment:
Isn't there a better way to do this? a more specific selector maybe? we should avoid `important` as much as we can.